src/content/eol/2024/04/eol-04-22-24.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "End of life for CPM and legacy synthetic monitoring runtimes"
+summary: "On October 22, 2024, New Relic will end of life (EOL) the containerized private minion (CPM) capability and legacy synthetic monitoring runtimes, including our legacy Chrome 72 (or older) and Node version 10 (or older) runtimes."
+publishDate: '2024-04-22'
+eolEffectiveDate: '2024-10-22'
+---
+
+On October 22, 2024, New Relic will end of life (EOL):
+
+* The Containerized Private Minion (CPM) capability
+* Legacy synthetic monitoring runtimes, including our legacy Chrome 72 (or older) and Node version 10 (or older) runtimes 
+
+## What you need to do
+
+Update to the latest runtimes by [following procedures in our transition guide](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime/), or by contacting support.
```

Comment:
Looks like this and other links need to be made relative

Reply:
Ah! I thought it was like what's new and we needed to make them absolute

Reply:
Good point! I actually don't now, can you test/check with Eng?

Reply:
EOL page links need to be absolute